File: benchmark/results/java-invocation-script/run-004/b-baseline/output/com/metaengine/demo/shipping/services/CarrierGateway.java
package com.metaengine.demo.shipping.services;

import com.metaengine.demo.shipping.aggregates.Shipment;
import java.util.List;

/** CarrierGateway integration service in the shipping domain. */
public class CarrierGateway {
    /** Dispatch a Shipment via the carrier from a partial input. */
    public Shipment create(Shipment input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a Shipment by id, or null if not found. */
    public Shipment findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Shipments up to limit. */
    public List<Shipment> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a Shipment by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-invocation-script/run-002/a-mcp/output/shipping/services/carrier_gateway.py
from typing import Optional
from shipping.aggregates.shipment import Shipment

"""CarrierGateway service."""
class CarrierGateway:
    def create(self, input: Shipment) -> Shipment:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Optional[Shipment]:
        raise NotImplementedError('not implemented')
    def list(self, limit: float) -> list[Shipment]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

